semgrep_output_v1.atd

```diff
@@ -816,6 +826,7 @@ type ci_scan_results <ocaml attr="deriving show"> = {
    searched_paths: string list;
    renamed_paths: string list;
    rule_ids: string list;
+   ?contributions: contributions option;
```

Comment:
We usually try to say since which version those additional fields have been supported, like
 (* since semgrep 1.22.0 *)
for example.

Reply:
I optimistically set it for 1.34.0. (I figure I can go back and update it if the semgrep-core code doesn't make it into the next release)